conans/client/conf/__init__.py

```diff
@@ -22,12 +22,14 @@
         version: ["7.0", "7.1", "8.0", "8.1", "8.2", "8.3", "9.0", "9.1", "9.2", "9.3", "10.0", "10.1", "10.2", "10.3"]
     FreeBSD:
     SunOS:
-arch: [x86, x86_64, ppc64le, ppc64, armv6, armv7, armv7hf, armv8, sparc, sparcv9, mips, mips64]
+    Arduino:
+        board: ANY
```

Comment:
While this might work, there could be a small problem with this: spelling. As ANY doesn't constraint settings, binaries for the same library for "Uno", "UNO", "ArduinoUNO", etc. will be different packages. The same for all boards.

It will also be a problem for the toolchains, that will have to manage these settings differently. Maybe, the definition of some new common settings for the wider community requires at least to provide a working toolchain for them. I am not sure how to approach it, it has always been challenging to model the embedded targets.